site/src/pages/release-notes/1.29.1.mdx

```diff
@@ -0,0 +1,25 @@
+---
+date: 2024-06-28
+---
+
+## 🛠️ Bug fixes
+
+- The <type://ExchangeType> is now correctly detected for the default <type://WebClient>. #5787
+- A duplicate key exception isn't raised anymore when building a JSON schema. #5788
+- The duplicator's child stream subscriber methods are now called by the correct executor. #5783
+
+## ☢️ Breaking changes
+
+- The signatures of <type://GrpcExceptionHandlerFunction#apply(RequestContext,Status,Throwable,Metadata)?full>
```

Comment:
Would you mind writing what problem is fixed by this change?

Reply:
Thanks! updated. 😉